LayoutTests/fast/speechsynthesis/speech-synthesis-voiceschanged-gc.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+</head>
+<body>
+<script>
+    description("This tests SpeechSynthesis can fire voiceschanged event without crash.");
+
+    if (window.internals)
+        window.internals.enableMockSpeechSynthesizer();
+
+    window.jsTestIsAsync = true;
+    if (window.testRunner)
+        testRunner.waitUntilDone();
+
+    const voices = speechSynthesis.getVoices();
+    speechSynthesis.onvoiceschanged = () => { finishJSTest(); };
```

Comment:
We should probably add an expando property on speechSynthesis or store speechSynthesis in a [WeakRef](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakRef) and check that the wrapper is identical before/after GC.